Library/OptimizelyClientType.swift

```diff
@@ -56,7 +56,7 @@ public func optimizelyTrackingAttributesAndEventTags(
 }
 
 public func optimizelyUserAttributes(
-  with user: User? = nil,
+  with user: User? = AppEnvironment.current.currentUser,
```

Comment:
Yeah was thinking this probably doesn't need to be an argument, we can just reference the environment directly. Also passing in `isAdmin` in the `variant` function is probably unnecessary. But we can address separately.